plugins/bazaar/src/util/sortMethods.ts

```diff
@@ -35,6 +35,15 @@ export const sortByTitle = (a: BazaarProject, b: BazaarProject) => {
   return 0;
 };
 
+export const sortByDecendingTitle = (a: BazaarProject, b: BazaarProject) => {
+  if (a.title < b.title) {
```

Comment:
comparing strings using `<` or `>` operators might produce wrong results, I would suggest another approach

Reply:
@vinzscam please review have modified the logic

Reply:
nice, thank you for fixing the other method as well